inst/templates/_targets.R

```diff
@@ -0,0 +1,47 @@
+library(targets)
+library(tarchetypes)
+
+## Load R files
+lapply(list.files("./R", pattern = ".R", full.names = TRUE), source)
+
+## A fix for the SRE to handle
+library(digest, warn.conflicts = FALSE)
+suppressPackageStartupMessages(library(R.utils, warn.conflicts = FALSE))
+.custom_check_file <- function (object, errormode) {
+  if (!file.exists(object)) {
+    return(.errorhandler("The file does not exist: ",
+                         object, mode = errormode))
+  }
+  if (!isTRUE(!file.info(object)$isdir)) {
+    return(.errorhandler("The specified pathname is not a file: ",
+                         object, mode = errormode))
+  }
+  if (suppressWarnings(R.utils::fileAccess(object, 4))) {
+    return(.errorhandler("The specified file is not readable: ",
+                         object, mode = errormode))
+  }
+}
+
+reassignInPackage("check_file", "digest", .custom_check_file)
```

Comment:
Should this only run conditionally if we are in the SRE? I think we can use this function outside as well?

Reply:
Yes good point. The only thing I thought about is maybe this could be a project .Rprofile so as not to clutter up the `targets.R` file. What about that idea?

Reply:
Great idea! So write the Rprofile if inside? Could also add library calls to usethis, devtools, etc?

Reply:
Worry a little about adding those library calls as it might interfere with the reassigning we are doing above.